EnvVars/newrelic.cmd

```diff
@@ -36,7 +39,7 @@ IF %NR_ERROR_LEVEL% EQU 0 (
 	:: WEB ROLES : Restart the service to pick up the new environment variables
 	:: 	if we are in a Worker Role then there is no need to restart W3SVC _or_
 	:: 	if we are emulating locally then do not restart W3SVC
-	IF "%IsWorkerRole%" EQU "false" IF "%EMULATED%" EQU "false" (
+	IF "%IsWorkerRole%" EQU "false" (

```

Comment:
Missed that, sorry. I'd still take a second to clean up this comment but not critical.